src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -138,6 +140,12 @@ public static bool IsTestMode
         /// </summary>
         public static bool EnableMigrationLogging { get; set; }
 
+        /// <summary>
+        /// An IRenderPackageFactory implementation which is used to generate
+        /// render packages.
+        /// </summary>
+        public static IRenderPackageFactory RenderPackageFactory { get; private set; }
+
```

Comment:
You need to make a strong case that this must be a static property.  I'm really not seeing why.  

It's so odd that you have to create a new `DynamoModel` to set this property...